plugins/kafka-backend/src/config/ClusterReader.ts

```diff
@@ -21,6 +21,7 @@ export function getClusterDetails(config: Config[]): ClusterDetails[] {
   return config.map(clusterConfig => {
     const clusterDetails = {
       name: clusterConfig.getString('name'),
+      alias: clusterConfig.getString('alias'),
```

Comment:
Was this intended for the Kafka backend?

Reply:
Kafka backend was by mistake, I will remove it. Only k8s.